yt/yt/server/lib/exec_node/config.cpp

```diff
@@ -763,6 +763,30 @@ void TNbdConfig::Register(TRegistrar registrar)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void TJobProxyLoggingConfig::Register(TRegistrar registrar)
+{
+    registrar.Parameter("mode", &TThis::Mode)
+        .Default();
+
+    registrar.Parameter("directory", &TThis::Directory)
```

Comment:
I think we should add postprocessing here and check that `Directory` is specified if `Mode` is `PerJobDirectory`